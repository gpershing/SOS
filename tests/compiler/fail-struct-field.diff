diff: ./tests/compiler/tests/compiler/fail-struct-field.err: No such file or directory
