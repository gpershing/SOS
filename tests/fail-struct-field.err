Fatal error: exception Failure("Could not find field y")
