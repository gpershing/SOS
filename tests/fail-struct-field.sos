struct a = {int x}

a s = {1}

int b = s.y
