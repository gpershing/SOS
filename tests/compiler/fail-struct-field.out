Could not find field y 
